governance/sig-governance.md

```diff
@@ -0,0 +1,81 @@
+# SIG Governance
+
+This charter details on the governance guidelines of TiDB SIGs. It will be updated as needed to meet the current needs of the TiDB Cimmunity.
+
+## Organizational Governance
+
+In order to standardize Special Interest Group efforts, create maximum transparency, and route contributors to the appropriate SIG, SIGs should follow these guidelines:
+
+- Create a charter and have it approved according to the SIG charter  process. Please refer [Create the SIG charter](#create-the-sig-charter) for more details.
+- Meet regularly, at least for 30 minutes every 2 weeks, except for long holidays
+- Keep up-to-date meeting notes, linked from the SIG's page in the community repo
+- Report activity in the weekly community meeting at least once every quarter
+- Participate in release planning meetings and retrospectives, and burndown meetings, as needed
+- Ensure related work happens in a project-owned github org and repository, with code and tests explicitly owned and supported by the SIG, including issue triage, PR reviews, test-failure response, bug  fixes, etc.
+- The assignable tasks will be graded by difficulty levels and claimed by SIG members.
+- Use the GitHub PR & issues, slack channels as the primary means of working, communicating, and collaborating.
+
+## SIG Roles
+
+The SIG follows the role definitions in the [TiDB Developer Group](/architecture/README.md#tidb-developer-group) and [GOVERNANCE](/GOVERNANCE.md).
+
+### SIG Members
+
+- Inludes all Contributors, Active Contributors, Reviewers, and Committers under the SIG
+- SHOULD maintain health of the SIG
+- SHOULD show sustained contributions to the SIG
+- SHOULD hold corresponding responsibilities in the SIG as documented in [TiDB Developer Group](/architecture/README.md#tidb-developer-group)
+- MAY participate in decision making of the SIG
+- Members are tracked in `membership.md` in the specific SIG (Contributors are listed in the aggregated contributor list for maintainence convenience)
+
+### Tech Lead
+
+  - **Number**: 2-3. Initially assigned by Maintainers
+  - **Requirement**:
+    - MUST be a committer from the specific SIG
+    - MUST have demonstrated deep understanding of the SIG project
+  - **Term**: 1 year
+  - **Election:**
+      - Nominated by the Tech Leads or self-nominate
+      - Gained supermajority votes of Maintainers
+      - Done through PR to update the `membership.md` file in the SIG
+- **Responsibilities**
+    - Creating new projects under the SIG
+    - Triaging issues/tasks in the SIG
+    - Mentoring members of the SIG
+    - Organizing discussions and drawing conclusions for decision making within the SIG
+    - Ensuring health and sustained development of the SIG and grwoth of SIG members
+    - Making roadmap for the SIG
```

Comment:
maybe we can remove it for now. currently, we only have a roadmap for the whole tidb product.